WalletWasabi.Fluent/Helpers/StartupHelpers/LinuxStartupHelper.cs

```diff
@@ -30,11 +30,11 @@ public static async Task AddOrRemoveDesktopFileAsync(bool runOnSystemStartup)
 					"Terminal=false",
 					"X-GNOME-Autostart-enabled=true");
 
-				await File.WriteAllTextAsync(pathToDesktopFile, fileContents).ConfigureAwait(false);
+				File.WriteAllText(PathToDesktopFile, fileContent);
 			}
 			else
 			{
-				File.Delete(pathToDesktopFile);
+				File.Delete(PathToDesktopFile);
```

Comment:
Use the magic IoHelper.Delete

Reply:
AFAIK `IoHelpers` class only got a `TryDeleteDirectoryAsync`.
Couldn't find `TryDeleteFile` or anything similar to that. Should I create one?

Reply:
No, I was wrong - this is fine as it is.